Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -361,16 +363,17 @@ final class CodecTests: MongoSwiftTestCase {
         {
             "double" : 2.0,
             "string" : "hi",
-            "doc" : { "x" : 1 },
+            "doc" : { "x" : { "$numberLong": "1" } },
```

Comment:
unfortunately, libmongoc's relaxed extJSON parser reads integers as `Int32`, so I needed to update all these relaxed instances to be canonical.